.build-emscripten.sh

```diff
@@ -36,8 +36,9 @@ pushd src/h3lib/lib
 cp ../../../../build/sizes.h ../include
 cp ../../../../build/sizes.c .
 # Compile with emscripten
-emcc -O3 -I ../include *.c -o libh3.js -DH3_HAVE_VLA -s INVOKE_RUN=0 -s EXPORT_NAME="'libh3'" -s MODULARIZE=1 -s NO_FILESYSTEM=1 -s NODEJS_CATCH_EXIT=0 -s TOTAL_MEMORY=33554432 -s ALLOW_MEMORY_GROWTH=1 -s WARN_UNALIGNED=1 -s EXPORTED_FUNCTIONS=$bound_functions -s EXTRA_EXPORTED_RUNTIME_METHODS='["cwrap", "getValue", "setValue"]' --memory-init-file 0
+emcc -O3 -I ../include *.c -o libh3.js -DH3_HAVE_VLA -s WASM=0 -s INVOKE_RUN=0 -s EXPORT_NAME="'libh3'" -s MODULARIZE=1 -s NO_FILESYSTEM=1 -s NODEJS_CATCH_EXIT=0 -s TOTAL_MEMORY=33554432 -s ALLOW_MEMORY_GROWTH=1 -s WARN_UNALIGNED=1 -s EXPORTED_FUNCTIONS=$bound_functions -s EXTRA_EXPORTED_RUNTIME_METHODS='["cwrap", "getValue", "setValue"]' --memory-init-file 0
 cp libh3.js ../../../../out/libh3.js
 echo "module.exports = libh3();" >> ../../../../out/libh3.js
+echo "Object.keys(module.exports.asmLibraryArg).filter(function(k) { return typeof module.exports.asmLibraryArg[k] === 'function'; }).forEach(function(k) { module.exports[module.exports.asmLibraryArg[k].name] = module.exports.asmLibraryArg[k]; });" >> ../../../../out/libh3.js
```

Comment:
For my info - why is this necessary now? Can you please add a comment explaining what's going on?

This might be cleaner assigning `module.exports` at the end, using arrow functions, and doing only one file append:
```
echo "const h3 = libh3(); Object.keys(h3.asmLibraryArg).filter(k => typeof h3.asmLibraryArg[k] === 'function').forEach(k => { h3[h3.asmLibraryArg[k].name] = h3.asmLibraryArg[k]; });" >> ../../../../out/libh3.js
```
Might be even nicer with heredoc:
```
cat << EOF >> ../../../../out/libh3.js
const h3 = libh3();
Object.keys(h3.asmLibraryArg)
  .filter(k => typeof h3.asmLibraryArg[k] === 'function')
  .forEach(k => {
    h3[h3.asmLibraryArg[k].name] = h3.asmLibraryArg[k];
  });
EOF
```

Reply:
So the latest version of Emscripten assumes that you're using WASM and not Asm.js and has dropped access to these helper functions from the Asm.js side of things (accidentally, I believe). I was planning to register an issue with Emscripten about this problem and reference this PR.

I found the methods still existing for use by the Asm.js code itself, but not for code interfacing with the Asm.js code and this reattaches these methods as they were before.

Your suggestions on how to format the code in the script are definitely good ones that I'll incorporate, though. :)

Reply:
Looks good, please add an explanatory comment here and then I think we're good to merge.